v2/internal/testcommon/test_context.go

```diff
@@ -20,18 +20,17 @@ import (
 	"testing"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
```

Comment:
Don't reorder this files imports. If you're using GoLand consider setting your imports options like this:

![image](https://user-images.githubusercontent.com/8798850/197900950-b0fa82d8-35db-4fa2-9e13-b774abb47440.png)

That matches the options we use in ASO and AKS.

Reply:
Thanks for this, Have updated.